src/test/java/teammates/test/cases/common/StringHelperTest.java

```diff
@@ -208,7 +208,18 @@ public void testRecoverFromSanitizedText(){
         str = Sanitizer.sanitizeForHtml("<text><div> 'param' &&& \\//\\");
         assertEquals("<text><div> 'param' &&& \\//\\",StringHelper.recoverFromSanitizedText(str));
     }
-    
+
+    @Test
+    public void testCountingEmptyStrings() {
+        String empty = "";
```

Comment:
Not considered empty. I'm not changing the behavior compared to the previous code. Will put that in the Javadoc.